editor/src/components/titlebar/title-bar.tsx

```diff
@@ -135,108 +136,101 @@ export const TitleBarProjectTitle = React.memo((props: { panelData: StoredPanel
+          width: '100%',
+          backgroundColor: theme.inspectorBackground.value,
+          paddingLeft: 10,
+          display: 'flex',
+          flexDirection: 'row',
+          alignItems: 'center',
+          gap: 10,
+          flexShrink: 0,
+        }}
+      >
+        <FlexRow css={{ gap: 6 }}>
+          <div
             style={{
-              gap: 5,
-              scale: hasUpstreamChanges || hasMergeConflicts || hasDownstreamChanges ? '75%' : 1,
-              transformOrigin: 'left',
+              width: 8,
+              height: 8,
+              borderRadius: 8,
+              backgroundColor: theme.unavailableGrey.value,
             }}
-          >
-            {repoName}
-            {<Icons.Branch style={{ width: 19, height: 19 }} />}
-            {currentBranch}
-          </SimpleFlexRow>
-        ) : (
-          <ProjectTitle>{projectName}</ProjectTitle>
-        )}
-        {when(
-          loggedIn,
+          />
           <div
             style={{
-              display: 'flex',
-              flexDirection: 'row',
-              gap: 6,
-              scale: '75%',
-              transformOrigin: 'left',
+              width: 8,
+              height: 8,
+              borderRadius: 8,
+              backgroundColor: theme.unavailableGrey.value,
             }}
-          >
-            {when(
-              hasUpstreamChanges,
-              <RoundButton
-                color={colorTheme.secondaryOrange.value}
-                onClick={openLeftPaneltoGithubTab}
-                onMouseDown={onMouseDown}
-              >
-                {<Icons.Download style={{ width: 19, height: 19 }} color={'on-light-main'} />}
-                <>Remote</>
-              </RoundButton>,
-            )}
-            {when(
-              hasMergeConflicts,
-              <RoundButton
-                color={colorTheme.error.value}
-                onClick={showMergeConflict}
-                onMouseDown={onMouseDown}
-              >
-                {
-                  <Icons.WarningTriangle
-                    style={{ width: 19, height: 19 }}
-                    color={'on-light-main'}
-                  />
-                }
-                <>Merge Conflicts</>
-              </RoundButton>,
-            )}
-            {when(
-              hasDownstreamChanges,
-              <RoundButton
-                color={colorTheme.secondaryBlue.value}
-                onClick={openLeftPaneltoGithubTab}
-                onMouseDown={onMouseDown}
-              >
-                {<Icons.Upload style={{ width: 19, height: 19 }} color={'on-light-main'} />}
-                <>Local</>
-              </RoundButton>,
-            )}
-          </div>,
-        )}
+          />
+        </FlexRow>
+        <div style={{ display: 'flex', flexDirection: 'column' }}>
+          {currentBranch != null ? (
+            <>
+              <SimpleFlexRow style={{ fontWeight: 600 }}>{currentBranch}</SimpleFlexRow>
```

Comment:
Can we add the branch icon next to this please? Or does it look cluttered with that?

Reply:
Complete redesign incoming...